in_toto/verifylib.py

```diff
@@ -1048,8 +1050,10 @@ def verify_disallow_rule(rule, source_artifacts_queue):
       source_artifacts_queue, rule_data["pattern"])
 
   if len(matched_artifacts):
-    raise RuleVerificationError("Rule '{0}' failed, pattern matched disallowed"
-        " artifacts: '{1}' ".format(" ".join(rule), matched_artifacts))
+    raise RuleVerificationError("Rule '{0}' failed.\nRule pattern matches the"
+        " following artifacts of the artifact queue, which is disallowed:\n"
+        " '{1}' \n'{2}'".format(" ".join(rule), matched_artifacts,
```

Comment:
Can you remove the single quotes "`'`" in `'{1}'` and `'{2}'`, they look a bit weird in the output.